modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/podcast/PlaylistUpdateAnalytics.kt

```diff
@@ -0,0 +1,236 @@
+            // we don't want to send update events
+            !playlist.draft -> sendPlaylistUpdateEvent(userPlaylistUpdate)
+        }
+    }
+
+    private fun sendPlaylistCreatedEvent(playlist: Playlist) {
+        val properties = buildMap<String, Any> {
+
+            put(Key.ALL_PODCASTS, playlist.allPodcasts)
+            colorAnalyticsValue(playlist)?.let {
+                put(Key.COLOR, it)
+            }
+            put(Key.DOWNLOADED, playlist.downloaded)
+            put(Key.DURATION, playlist.filterDuration)
+            put(Key.EPISODE_STATUS_IN_PROGRESS, playlist.partiallyPlayed)
+            put(Key.EPISODE_STATUS_PLAYED, playlist.finished)
+            put(Key.EPISODE_STATUS_UNPLAYED, playlist.unplayed)
+            iconAnalayticsValue(playlist)?.let {
+                put(Key.ICON_NAME, it)
+            }
+            mediaTypeAnalyticsValue(playlist)?.let {
+                put(Key.MEDIA_TYPE, it)
+            }
+            put(Key.STARRED, playlist.starred)
+            releaseDateAnalyticsValue(playlist)?.let {
+                put(Key.RELEASE_DATE, it)
+            }
+        }
+
+        analyticsTracker.track(AnalyticsEvent.FILTER_CREATED, properties)
+    }
+
+    private fun iconAnalayticsValue(playlist: Playlist) =
+        when (playlist.drawableId) {
+            IR.drawable.ic_filters_list,
+            IR.drawable.auto_filter_list,
+            IR.drawable.automotive_filter_list -> Value.IconName.LIST
+
+            IR.drawable.ic_filters_headphones,
+            IR.drawable.auto_filter_headphones,
+            IR.drawable.automotive_filter_headphones -> Value.IconName.HEADPHONES
+
+            IR.drawable.ic_filters_clock,
+            IR.drawable.auto_filter_clock,
+            IR.drawable.automotive_filter_clock -> Value.IconName.CLOCK
+
+            IR.drawable.ic_filters_download,
+            IR.drawable.auto_filter_downloaded,
+            IR.drawable.automotive_filter_downloaded -> Value.IconName.DOWNLOADED
+
+            IR.drawable.ic_filters_play,
+            IR.drawable.auto_filter_play,
+            IR.drawable.automotive_filter_play -> Value.IconName.PLAY
+
+            IR.drawable.ic_filters_volume,
+            IR.drawable.auto_filter_volume,
+            IR.drawable.automotive_filter_volume -> Value.IconName.VOLUME
+
+            IR.drawable.ic_filters_video,
+            IR.drawable.auto_filter_video,
+            IR.drawable.automotive_filter_video -> Value.IconName.VIDEO
+
+            IR.drawable.ic_filters_star,
+            IR.drawable.auto_filter_star,
+            IR.drawable.automotive_filter_star -> Value.IconName.STARRED
+
+            else -> {
+                Timber.e("No matching analytics icon found")
+                null
+            }
+        }
+
+    private fun mediaTypeAnalyticsValue(playlist: Playlist) =
+        when (playlist.audioVideo) {
+            0 -> Value.MediaType.ALL
```

Comment:
```suggestion
            Playlist.AUDIO_VIDEO_FILTER_ALL -> Value.MediaType.ALL
```
Should this use this constant instead?

Reply:
Yes, I updated to use all AUDIO_VIDEO_FILTER constants in [ede79da](https://github.com/Automattic/pocket-casts-android/pull/414/commits/ede79da6905e3ae595b187d6b98e8a53d6e37b05). That's much better, thanks!